README.md

```diff
@@ -1 +1,41 @@
-# listview-filter
\ No newline at end of file
+[![Build Status](https://travis-ci.org/mendixlabs/listview-filter.svg?branch=master)](https://travis-ci.org/mendixlabs/listview-filter)
+[![Dependency Status](https://david-dm.org/mendixlabs/listview-filter.svg)](https://david-dm.org/mendixlabs/listview-filter)
+[![Dev Dependency Status](https://david-dm.org/mendixlabs/listview-filter.svg#info=devDependencies)](https://david-dm.org/mendixlabs/listview-filter#info=devDependencies)
+[![codecov](https://codecov.io/gh/mendixlabs/listview-filter/branch/master/graph/badge.svg)](https://codecov.io/gh/mendixlabs/listview-filter)
+
+# Drop-down filter
+Filter mendix list views using a drop-down
```

Comment:
`Enable users to filter a list view at run time, with various options that can be selected from a drop-down`

Reply:
Done